src/cars.js

```diff
@@ -0,0 +1,10 @@
+function maxByPosition(cars) {
+  return Math.max(...cars.map((it) => it.position));
+}
+
+function filterByPosition(cars, position) {
+  return cars.filter((it) => it.position === position);
+}
+
+module.exports.maxByPosition = maxByPosition;
+module.exports.filterByPosition = filterByPosition;
```

Comment:
ES6 이후부터는 [단축 속성명](https://ahnheejong.gitbook.io/ts-for-jsdev/02-ecmascript/object-and-array/object-literal-diff#shorthand-property-name)을 사용할 수 있어 요 export 구문에도 동일하게 적용할 수 있습니다.      
요런 방식은 어떠신가요? 🙂 
```suggestion
module.exports = {
    maxByPosition,
    filterByPosition
};
```